src/Channel.ts

```diff
@@ -242,9 +243,10 @@ export default class Channel {
       tileoffset = tilex * x + tiley * y * atlasrow;
       for (let j = 0; j < y; ++j) {
         tilerowoffset = j * atlasrow;
-        for (let k = 0; k < x; ++k) {
-          volimgdata[tileoffset + tilerowoffset + k] = this.volumeData[i * (x * y) + j * x + k];
-        }
+        volimgdata.set(
+          this.volumeData.subarray(i * (x * y) + j * x, i * (x * y) + j * x + x),
+          tileoffset + tilerowoffset
+        );
```

Comment:
This is the code that turns volume arrays into a texture atlased 2d layout for rendering.  The ome-zarr code path goes through this function.

Reply:
could/should `i * (x * y) + j * x` be factored out to make the arguments to `subarray` something like `(offset, offset + x)`?

Reply:
I have no problem with that... I'll do it